ilastik/applets/dataSelection/datasetDetailedInfoTableView.py

```diff
@@ -418,9 +419,18 @@ def dragMoveEvent(self, event):
 
     def dropEvent(self, dropEvent):
```

Comment:
Maybe this could be simplified?

```python
def dropEvent(self, dropEvent):
    filepaths = [Path(QUrl.toLocalFile(url)) for url in dropEvent.mimeData().urls()]
    starting_lane_index = self.rowAt(dropEvent.pos().y())
    # Last row is the button.
    if starting_lane_index == self.model().rowCount() - 1:
        starting_lane_index = -1
    self.addFilesRequestedDrop.emit(filepaths, starting_lane_index)
```